docs/4.3/admin-guide.md

```diff
@@ -0,0 +1,2622 @@
+# Teleport Admin Manual
+
+This manual covers the installation and configuration of Teleport and the
+ongoing management of a Teleport cluster. It assumes that the reader has good
+understanding of Linux administration.
+
+## Installing
+
+To install, download the official binaries from the [Teleport
+Downloads](https://gravitational.com/teleport/download/) section on our web site
+and run:
+
+```
+$ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ sudo make install
+```
+
+### Installing from Source
+
+Gravitational Teleport is written in Go language. It requires Golang v{{ teleport.golang }} or
+newer.
+
+``` bash
+# get the source & build:
+$ mkdir -p $GOPATH/src/github.com/gravitational
+$ cd $GOPATH/src/github.com/gravitational
+$ git clone https://github.com/gravitational/teleport.git
+$ cd teleport
+$ make full
+
+# create the default data directory before starting:
+$ sudo mkdir -p /var/lib/teleport
+```
+
+### Teleport Checksum
+
+Gravitational Teleport provides a checksum from the
+[Downloads](https://gravitational.com/teleport/download/) page. This can be
+used to verify the integrity of our binary.
+
+Example using v4.0.8 *(replace v4.0.8 with the version of Teleport you are using)*:
```

Comment:
Should we `{{ teleport.version }}` these mentions of 4.0.8 too so that they stay up to date?

Reply:
I'll fixed this after we publish and update the screenshot.